### ISSUE TITLE
Effect during transition

### ISSUE THREAD

**Issue by bctiemann** [kthornbloom/Smoothslides#27]
Hi—nice work! I noticed however that the panning (or other effects) comes to a stop before the crossfade transition between slides. Usually a "ken burns" style slideshow of this type would allow the slides to continue panning over each other throughout the crossfade between them. Is this a possibility?

Thanks!

**Comment by kthornbloom** [comment]
Hey there. I agree that would be better! The issue I ran into is that I'm running a CSS transition for those panning effects. I can't start a new transition (opacity for the fade-out) while the original one is still going without using a bunch of keyframes or something. I suppose you might able to do the fade on a wrapper element rather than the one animating... but you might have performance problems there.

Long story short, it doesn't have that ability at the moment but I think it sounds nice :)

**Comment by bctiemann** [comment]
Cool--this one seems to do it, maybe you can take a look at how it's doing it?

http://cssmojo.com/ken-burns-effect/

I like your implementation of the pluggable extension (one of my needs is to have multiple slideshows on the page), and this would make it pretty much ideal. Thanks!
